StratisCore.UI/main.ts

```diff
@@ -56,7 +56,7 @@ const coreargs = require('minimist')(args, {
 	port: portDefault,
 	signalrport: 38823,
 	rpcServer: 1,
-	rpcallowip: '0.0.0.0/0',
+	rpcallowip: '127.0.0.0/8',
```

Comment:
Why do we change this?

Reply:
I don't know anything about this change, as discussed will revert.